deployments/root/conda-envs/alchemiscale-client.yml

```diff
@@ -1,27 +1,31 @@
 name: alchemiscale-client
 channels:
-  - jaimergp/label/unsupported-cudatoolkit-shim
   - conda-forge
 
 dependencies:
   - pip
-  - python=3.12
+  - python =3.12
+  - cuda-version >=12
 
-  - alchemiscale-client=0.5.3
+  - alchemiscale-client =0.6.0
 
   # openmm protocols
-  - feflow=0.1.1
+  - feflow =0.1.2
 
   # additional pins
-  - gufe=1.1.0
-  - openfe=1.2.0
-  - openmm=8.1.2
-  - openmmforcefields>=0.14.1
-  - openff-units=0.2.2
+  - gufe =1.2.0
+  - openfe =1.3.0
+  - openmm =8.1.2
+  - openmmforcefields >=0.14.1
+  - openff-units =0.2.2
+
+  ## pins due to breaking changes
+  - openff-interchange =0.4.1    # https://github.com/openmm/openmmforcefields/issues/365
 
   # alchemiscale-fah dependencies
   - cryptography
   - plyvel
 
   - pip:
-    - git+https://github.com/OpenFreeEnergy/alchemiscale-fah.git@v0.1.1
+    - git+https://github.com/OpenFreeEnergy/alchemiscale-fah.git@v0.1.2
+    - git+https://github.com/OpenFreeEnergy/pontibus.git@v0.0.1
```

Comment:
Just FYI, there is a pontibus conda-forge recipe out now. However, it includes nagl-base which should be optional and might be too large for alchemiscale.

Reply:
Let's start with this, and if we hit issues we'll consider using the conda package next.